test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -74,7 +74,7 @@ public void Muxer_Exec_activation_of_Build_Output_Portable_DLL_with_DepsJson_Loc
             dotnet.Exec("exec", "--runtimeconfig", runtimeConfig, appDll)
                 .CaptureStdErr()
                 .CaptureStdOut()
-                .Execute()
+                .Execute(true)
```

Comment:
or still better would be to  use, the pattern followed in sdk and cli repo's fluent style

```C#
.Execute
.ShouldFail()
```